diesel/src/sqlite/backend.rs

```diff
@@ -50,4 +50,7 @@ impl TypeMetadata for Sqlite {
     type MetadataLookup = ();
 }
 
+impl SupportsDefaultKeyword for Sqlite {}
+impl SupportsReturningClause for Sqlite {}
```

Comment:
Sqlite does not support the default key word. Also returning clauses are not supported. So both traits should not be implemented because otherwise invalid sql will be generated.